frontend/src/components/common/Resource/ConfirmButton.tsx

```diff
@@ -1,6 +1,6 @@
 import Button, { ButtonProps } from '@material-ui/core/Button';
 import React from 'react';
-import { ConfirmDialog } from './Dialog';
+import { ConfirmDialog } from '../Dialog';
```

Comment:
I don't think the ConfirmButton has something to do with a cluster resource. What am I missing?

Reply:
Yes, my mistake. Moved it back.